client/client.go

```diff
@@ -162,6 +175,15 @@ type Client struct {
 	// garbageCollector is used to garbage collect terminal allocations present
 	// in the node automatically
 	garbageCollector *AllocGarbageCollector
+
+	// aclCache is used to maintain the parsed ACL objects
```

Comment:
Can we create an object like `clientAclResolver` and put all this stuff in the `client/acl.go` file and move all the consts there too. Want to avoid the Client object become huge.